spec/controllers/api/shipments_controller_spec.rb

```diff
@@ -14,12 +15,12 @@
 
   context "as a non-admin" do
     it "cannot make a shipment ready" do
-      api_put :ready
+      api_put :ready, order_id: shipment.order.to_param, id: shipment.to_param
```

Comment:
thanks. This is the sort of change that will save ous valuable minutes while investigating a test failure :ok_hand: